yarn.lock

```diff
@@ -11837,7 +11837,7 @@ fsevents@^1.2.7:
     node-fetch: ~2.6.1
     numeral: ~2.0.6
     path-complete-extname: ~0.1.0
-    patternfly: ~3.59.1
+    patternfly: ~3.59.5
     patternfly-bootstrap-treeview: ~2.1.7
```

Comment:
I expected this to bump as well, since this was bumped in patternfly itself: https://github.com/patternfly/patternfly-3/commit/63a20c8c656dd883b3c6249f21249e6e7bc30621